promql/quantile.go

```diff
@@ -72,16 +72,17 @@ type metricWithBuckets struct {
 //
 // If q>1, +Inf is returned.
 //
-// We also return a bool to indicate if monotonicity needed to be forced.
-func bucketQuantile(q float64, buckets buckets) (float64, bool) {
+// We also return a bool to indicate if monotonicity needed to be forced,
+// and another bool to indicate if precision errors needed to be corrected.
```

Comment:
I'm a bit unhappy with the wording "precision errors ... corrected". We are doing something fairly blunt (considering values with small differences as equal), that's not really a "correction".

Maybe "... to indicate if small differences between buckets (that are likely artifacts of floating point precision issues) have been ignored." ?